components/brave_new_tab_ui/components/search/config.ts

```diff
@@ -26,7 +26,17 @@ const getConfig = () => {
   return cache!
 }
 
-export const setEngineEnabled = (engine: SearchEngineInfo, enabled: boolean) => {
+// Determines whether any search engines are enabled
+export const hasEnabledEngine = () => Object.values(getConfig()).find(c => c)
```

Comment:
q. could using `.length` be more readable(as long as it's what's intended)?

```suggestion
export const hasEnabledEngine = () => Object.values(getConfig()).length
```

Reply:
It would have to be `.filter(c => c).length` as the disabled search engines are still stored (as `false`)